tests/test_bot.py

```diff
@@ -0,0 +1,35 @@
+import time
+import pytest
+
+
+TIMEOUT = 60
+
+
+class TestAdminGroup:
+    @pytest.mark.timeout(TIMEOUT)
```

Comment:
i think you can apply the timeout on the class level -- see https://pytest-timeouts.readthedocs.io/en/latest/usage/
or maybe even at global level.

Reply:
global didn't work, class did.